### ISSUE TITLE
[FEEDBACK] - Point-to-point

### ISSUE THREAD

**Issue by jdttaylor** [tewhatuora/api-standards#131]
### Summary

This page needs some review. I think I understand what it is trying to say but someone new to EDA may get confused between pub/sub and point to point.

Fundamentally point-to-point is a one to one connection between a producer and a consumer. It is correct that you can have multiple consumers, but I would say they compete. If you are using a broker, the broker should support round robin, or fail over, however I think this is an advanced topic (not an event term) and will confuse the reader.

The example Immunization administered, its a little misleading, I'm assuming the consumer is AIR. If implementing the point to point pattern, there would be multiple producers, and AIR would need to establish a direct connection with each producer. Obviously a broker is in the mix which enables you to break the rules, but if you explain how a broker can simply the immunization administered use case, you are moving away from the point-to-point definition.

This sentence doesn't make sense.
_The messages are put onto a queue and consumers consume the messages, before acknowledging them as consumed where they are removed from the queue_

This sentence is not true, only once consumer will read the message, as point to point guarantees delivery, if multiple consumers process the same message there will be a high chance of duplicates / conflict management. If ordering is also important then multiple consumers causes complexity.
_If there is more than one consumer, they each read messages from the queue_
Also its a little confusing as you are explaining point-to-point is a one-to-one exchange. This advanced topic really needs to be introduced.

### Link to standards item

https://apistandards.digital.health.nz/draft/api-development/Asynchronous%20APIs/Async%20Patterns/PointToPoint

### Which area of the standards does this apply to?

- [ ] Part A - API Concepts
- [ ] Part B - API Security
- [X] Part C - API Design and Development
- [ ] Part D - FHIR
- [ ] Community guidelines

**Comment by kyle-mwnz** [comment]
Hi @jdttaylor thank you for the feedback - we are working on some updates to improve this page

**Comment by mathusmith** [comment]
This pattern is valid and relevant, but it would benefit from some refinement. Some additional comments for clarity
- One, or many producers can send messages to a message broker queue. One, or many consumers can access that queue. However, any message on that queue can be received by only one consumer. 
- Use point-to-point messaging when you want to ensure that a message is received by only one consumer application.
- If more than one consumer is required to consume the message, publish-subscribe should be used instead.

**Comment by kyle-mwnz** [comment]
Hi @jdttaylor and @mathusmith - some updates based on the feedback provided in this issue have been made to the point to point page on the Draft site. If you could take a look that would be great. Thanks

**Comment by swithinfoote** [comment]
Closing this issue - @jdttaylor please open a new issue if you would like further changes /clarifications.

**Comment by mathusmith** [comment]
Confirming that the revisions adequately resolve the issue.
